pkg/metrics/metrics.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The Kubernetes Authors.
+Copyright 2023 The Kubernetes Authors.
```

Comment:
This file is not updated.

Reply:
There are too many files needs to be changed. So create another PR to address it separately: 
https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/pull/209